src/nativestore/RelationBlock.cpp

```diff
@@ -240,8 +240,9 @@ RelationBlock* RelationBlock::get(unsigned int address) {
     int RECORD_SIZE = sizeof(unsigned int);
     if (address == 0) {
         return NULL;
-    } else if (address % RelationBlock::BLOCK_SIZE != 0) {
-        throw "Exception: Invalid relation block address !!\n received address = " + address;
+    }
+    if (address % RelationBlock::BLOCK_SIZE != 0) {
+        throw std::logic_error("Exception: Invalid relation block address !!\n received address = " + address);
```

Comment:
Rather than throwing this error shall we log it?

Reply:
Changed to log error

Reply:
Ok